testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabUserHelper.java

```diff
@@ -233,8 +235,13 @@ public static String loadUserForTest(LabUserQuery query) {
         }
 
         LabConfig.setCurrentLabConfig(labConfig);
+        UserInfo userInfo = labConfig.getConfigInfo().getUserInfo();
+        String upn = userInfo.getHomeUPN();
+        if (StringUtil.isNullOrEmpty(upn) || upn.equalsIgnoreCase("None")) {
+            upn = userInfo.getUpn();
+        }
 
-        return labConfig.getConfigInfo().getUserInfo().getHomeUPN();
+        return upn;
```

Comment:
Has something changed that is causing us to make this change? Why was it working all the way until now?

Reply:
We need to use the HomeUpn property for cross-cloud scenarios

Reply:
and looks like we were already using that, right? Now you've added a fallback to use the `upn` property if `homeUpn` is not there...what's the reason behind this?

Reply:
No, we are using upon currently.